roles/openshift_logging/defaults/main.yml

```diff
@@ -3,7 +3,7 @@ openshift_logging_image_prefix: docker.io/openshift/origin-
 openshift_logging_image_version: latest
 openshift_logging_use_ops: False
 master_url: "https://kubernetes.default.svc.{{ openshift.common.dns_domain }}"
-public_master_url: "https://{{openshift.common.public_hostname}}:8443"
+openshift_logging_master_public_url: "https://{{openshift.common.public_hostname}}:8443"
```

Comment:
edit - do we expect users to change this value given that we're pointing it to the public_hostname gathered by facts?